packages/selenium/src/Resources/finishRun.js

```diff
@@ -0,0 +1,5 @@
+const cb = arguments[arguments.length - 1];
+const partialResults = JSON.parse(arguments[0]).flat();
+const options = JSON.parse(arguments[1])
+axe.finishRun(partialResults, options)
+    .then((results) => cb(results));
```

Comment:
Does this need a `catch` handler to propagate `axe.finishRun` rejecting, like `legacyScan.js` handles?